File: Golang/409. Longest Palindrome.go
func longestPalindrome(s string) int {
    charDict := [82]int{}
    for _, char := range s {
        charDict[int(char - 'A')]++
    }
    
    ans := 0
    
    for i := range charDict {
        ans += (charDict[i] / 2) * 2
    }
    
    if ans < len(s) {
        return ans + 1
    }
    return ans
}

File: Python3/409. Longest Palindrome.py
class Solution:
    def longestPalindrome(self, s):
        """
        :type s: str
        :rtype: int
        """
        count = 0
        charDict = collections.Counter(s)
        for num in charDict.values():
            count += (num // 2)*2
        return count+1 if count < len(s) else count
